src/client/scripts/esm/game/input.js

```diff
@@ -113,6 +119,30 @@ function getMouseWorldLocation() {
 	return [mouseWorldLocation[0], mouseWorldLocation[1]];
 }
 
+function getPointerDown() {
+	return usingTouchscreen ? touchDowns.length === 1 : mouseDowns.includes(leftMouseKey);
+}
+
+function getPointerHeld() {
+	return usingTouchscreen ? touchHelds.length === 1 : mouseHelds.includes(leftMouseKey);
+}
+
+function getPointerClicked() {
+	return usingTouchscreen ? touchClicked : mouseClicked;
+}
+
+function getPointerClickedTile() {
+	return usingTouchscreen ? touchClickedTile : mouseClickedTile;
+}
+
+function getPointerWorldLocation() {
+	return [pointerWorldLocation[0], pointerWorldLocation[1]];
+}
+
+function getUsingTouchscreen() {
```

Comment:
Again, docutil provides a method for determining if the device is touch compatible, or is this function meant to just tell you whether the last pointer event was touch-activated?